shumai/stats/op_to_flops.ts

```diff
@@ -21,3 +23,12 @@ function opToFlopsMatmul(inputs: any[], out: Tensor): number {
 
   return k * out.elements * 2
 }
+
+function opToFlopsConv2d(inputs: any[], out: Tensor): number {
+  const [, w] = inputs
+
+  const [Co, Ci, Kw, Kh] = w.shape
+  const [, , Wo, Ho] = out.shape
+
+  return Kw * Kh * Ci * Wo * Ho * Co
```

Comment:
```suggestion
  return Kw * Kh * Ci * Wo * Ho * Co * 2
```

Reply:
Makes sense. I had seen some conflicting data online so wasn't certain, but based on my testing this seems correct.